src/datastore/database.hpp

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include <vector>
+#include "souvenir.hpp"
+#include "stadium.hpp"
+#include "team.hpp"
+
+/**
+ * @class Database class
+ *
+ * This class holds all of the programs data.
+ * It contains a map of teams and a map of stadiums.
+ * Each stadium has a vector of souvenirs within it's class.
+ *
+ * All the members aside from getEnumValue are static
+ */
+class Database
+{
+public:
+    /* Constructor */
+    Database();
+
+    /* Destructor */
+    ~Database();
+
+    /* Static methods to Load and Save from/to file */
+    static void loadFromFile(const std::string&);
+    static void saveToFile(const std::string&);
+
+     /* Getters */
+    static std::map<int,Team> getTeams();
```

Comment:
Is returning a map necessary? Why not just have a single getter for teams and stadiums that returns a vector?

Reply:
The maps was your idea, added the ability to return vectors for the team listings

Reply:
Marked as resolved because you changed your mind